scripts/globals/mobskills/brainjack.lua

```diff
@@ -8,9 +8,10 @@ require("scripts/globals/monstertpmoves");
 ---------------------------------------------------
 
 function onMobSkillCheck(target,mob,skill)
-    local mobSkin = mob:getModelId();
+    -- See PW's script
+    local phase = mob:getLocalVar("phase");
 
-    if (mobSkin == 1825) then
+    if (phase == 2) then
```

Comment:
Just a head sup about a few things:

I know the other mobs that uses these skills aren't in the project yet, but this is one of the reasons it needs to be handled as skill list swaps instead of a localvar or the previous modelId checks. 

its how the skill wound up with that dumb prefix_ because it used to be 2 scripts and when that changed one of them got deleted (since PW was "implemented" at the time its version is the one that stayed). This Pr is likely going to get conflicts with 2 others that are open. One of which is waiting on the BCNM branch I think. This one may wind up waiting awhile and then need some conflict resolution love to fix.